README.md

```diff
@@ -1 +1,33 @@
-# gallery
\ No newline at end of file
+# gallery
+
+Gallery provides Loom's UI for interacting with the entity data model, as well as downloading and visualizing datasets.
+
+**Setup**
+
+1. Install [cassandra](https://docs.datastax.com/en/cassandra/2.0/cassandra/install/installDeb_t.html)
+2. Clone the entire [loom](http://stash.krypt.int/projects/KRYP/repos/loom/browse) superproject
+3. Install dependencies.
```

Comment:
Clarifying with an "Install NPM dependencies" seems reasonable. I think the larger issue is that these instructions can only depend on publicly available projects, so I would just say git clone conductor,datastore, and gallery.  The other documentation can go in the loom submodules project which is still private.